backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/BpkTextField.kt

```diff
@@ -58,7 +63,55 @@ fun BpkTextField(
         modifier = modifier,
         readOnly = readOnly,
         placeholder = placeholder,
-        icon = icon,
+        prefix = {
```

Comment:
Open to suggestions for the name of this

Reply:
maybe the internal text field implementation can change to take the prefix sealed class from the search input summary, so we only need to decide in one place how to render an icon or text and keep it consistent